hub-templates/basehub/values.yaml

```diff
@@ -27,6 +27,9 @@ nfsPVC:
     # MUST HAVE TRAILING SLASH
     baseShareName: /export/home-01/homes/
 
+# In-cluster NFS storage
+nfsGanesha:
+  enabled: false
```

Comment:
I was thinking about a more explicit name than this. I miss two aspects captured in this naming:
- this influence 2i2c specific templates, not a subchart like `nfs-server-provisioner` running NFS ganesha
- this influence the creation of a PV resource specifically.

In practice I can't come up with good names without breaking misc patterns in how config has been named, but I think what I suggest is...


```suggestion
# Configuration related to integrating with the nfs-server-provisioner Helm chart that run
# nfs-ganesha and provides the custom "nfs" StorageClass. Together they provide in cluster
# NFS storage for PVs configured with a storageClassName of "nfs".
nfsGanesha:
  enabled: false
```